examples/error-type-intersection/error_type_intersection.md

```diff
@@ -1,6 +1,6 @@
 # Type intersection for error types
 
-You can define an error type that is both a subtype of a `distinct` error type and has additional constraints on the detail fields using intersection types.
+If you intersect two `error` types, the resulting type's detail type is the intersection of the detail types of both types. Furthermore, if any of the types being intersected is a distinct type, then the resultant type's type ID set includes all the type IDs of that type.
```

Comment:
I think we have to relate to it therefore being a subtype.

Reply:
And maybe emphasize that this way you can define a custom error with a specific detail type that is also a subtype of a distinct error type (this is the only way).

Reply:
Fixed